fill_templates.py

```diff
@@ -23,6 +23,16 @@
 )
 SETUP_DIR = os.path.dirname(os.path.abspath(__file__))
 PROPERTIES_DIR = os.path.join(SETUP_DIR, "fv3gfs/wrapper")
+FORTRAN_TO_C_TYPES = {
+    "integer": {"type_c": "c_int", "type_cython": "int"},
+    "real": {"type_c": "c_double", "type_cython": "REAL_t"},
+    "logical": {"type_c": "c_int", "type_cython": "bint"},
```

Comment:
Previously we used `"logical": {"type_c": "c_bool", "type_cython": "bint"}`.  With `"c_bool"` I found the results returned for boolean flags were incorrect (i.e. they did not reflect what was in the namelist and sometimes varied from rank to rank).  Using `"c_int"` appears to fix this.  

I'll admit though, I'm not expert on this interaction between fortran, C, and Python, so it is not clear to me why this is the case and whether this is the most correct fix.